.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
```

Comment:
Isort will automatically pickup ``.editorconfig`` so this file shouldn't be needed.